google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1035,6 +1035,62 @@ def public_access_prevention_inherited?
 
         alias public_access_prevention_unspecified? public_access_prevention_inherited?
 
+        ##
+        # Recovery Point Objective (RPO) is another metdata of a bucket, it measures how long it takes for a set of
+        # updates to be asynchronously copied to the other region.
+        # Currently, `DEFAULT` and `ASYNC_TURBO` are supported. When set to `ASYNC_TURBO`, Turbo Replication is enabled
+        # for a bucket. `DEFAULT` is used to reset rpo on an existing bucket with rpo set to `ASYNC_TURBO`.
+        # This value can be modified by calling {#rpo=}.
+        #
+        # @return [String, nil] Currently, `DEFAULT` and `ASYNC_TURBO` are supported. Returns `nil` if the bucket has
+        #    no RPO.
+        #
+        # @example
+        #   require "google/cloud/storage"
+        #
+        #   storage = Google::Cloud::Storage.new
+        #
+        #   bucket = storage.bucket "my-bucket"
+        #
+        #   bucket.rpo = :DEFAULT
+        #   bucket.rpo #=> "DEFAULT"
+        #
+        def rpo
+          @gapi.rpo
+        end
+
+        ##
+        # Sets the value for Recovery Point Objective (RPO) in the bucket. This value can be queried by calling {#rpo}.
+        #
+        # @param [Symbol, String] new_rpo The bucket's new Recovery Point Objective metadata.
```

Comment:
Probably because the code says `@gapi.rpo = new_rpo.to_s` which would attempt to set it to the empty string if you pass `nil` for `new_rpo`. What if you changed it to `@gapi.rpo = new_rpo&.to_s` to preserve nils?